src/platform/forms/components/common/grid/Column.jsx

```diff
@@ -0,0 +1,10 @@
+// libs
+import React from 'react';
+
+function Column(props) {
+  const { children, classNames = '' } = props;
+
+  return <div className={`${classNames} columns`}>{children}</div>;
+}
+
+export default React.memo(Column);

```

Comment:
Oh, cool. Didn't realize this was a thing. :+1: 

My understanding is that it'll only re-render when the props change, but I'm not sure how this is different from a standard functional component. What benefit do we gain here? (Asking to sate my own curiosity more than anything else.)

Reply:
Probably not the best place for them and going to remove for now. Maybe we revisit and layer in as needed on some of the larger components?